lib/protocol/networks.js

```diff
@@ -899,23 +897,23 @@ simnet.pow.targetTimespan =
   simnet.pow.targetWindow * simnet.pow.targetSpacing;
 simnet.pow.minActual = (simnet.pow.targetTimespan / 4) >>> 0;
 simnet.pow.maxActual = simnet.pow.targetTimespan * 4;
-simnet.pow.targetReset = false;
+simnet.pow.targetReset = true;
 simnet.pow.noRetargeting = false;
 
 simnet.names = {
   auctionStart: 0,
   rolloutInterval: 1,
   lockupPeriod: 1,
-  renewalWindow: 2500,
+  renewalWindow: simnet.pow.blocksPerDay,
   renewalPeriod: 1250,
-  renewalMaturity: 25,
+  renewalMaturity: 1,
   claimPeriod: 75000,
-  biddingPeriod: 25,
-  revealPeriod: 50,
+  biddingPeriod: simnet.pow.blocksPerDay,
+  revealPeriod: simnet.pow.blocksPerDay,
   treeInterval: 2,
   transferLockup: 5,
   revocationDelay: 25,
-  auctionMaturity: 25 + 50 + 25,
+  auctionMaturity: 2 + 5 + 25,
   noRollout: false,
```

Comment:
Following the same pattern here as in regtest.

`auctionMaturity` = `treeInterval` + `trasferLockup` + `revocationDelay`

https://github.com/handshake-org/hsd/blob/af86d4899791601d10212115c80a2f2136fdb004/lib/protocol/networks.js#L763-L766